easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.0.6-GCC-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '2.0.6'
+
+homepage = 'http://www.iqtree.org'
+description = "Efficient phylogenomic software by maximum likelihood "
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/Cibiv/IQ-TREE/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['535ca86c7655e68785c5efbfb020006ed7d7de0e2f17e9f31383202ff81f3a85']  # 2.0.6.tar.gz
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('Eigen', '3.3.9'),
+    ('Boost', '1.74.0'),  # apparently required (thanks Zao!)
```

Comment:
I don't think we need a comment things like this.

Also, I suspect it's not just a build dep.

Reply:
I removed the comment and moved BOOST to the dependencies. I think I got Zao here wrong.